### PR TITLE
[FIX] Missing height on departments agents table

### PR DESCRIPTION
**Pull request by dougfabris**
## Proposed changes (including videos or screenshots)

image

### CODE REVIEW

File: client/views/omnichannel/departments/DepartmentEdit.js [hunk 1]
```diff
@@ -347,7 +347,7 @@ export function EditDepartment({ data, id, title, reload }) {
 					<Divider mb='x16' />
 					<Field>
 						<Field.Label mb='x4'>{t('Agents')}:</Field.Label>
-						<DepartmentsAgentsTable agents={data && data.agents} setAgentListFinal={setAgentList}/>
+						<Box display='flex' flexDirection='column' height='50vh'><DepartmentsAgentsTable agents={data && data.agents} setAgentListFinal={setAgentList}/></Box>
```
Comment (tassoevan) [inline client/views/omnichannel/departments/DepartmentEdit.js:350]: Why not `minHeight`?

Comment (dougfabris) [reply client/views/omnichannel/departments/DepartmentEdit.js]: @tassoevan I agree, thanks!

**Comment by renatobecker** [comment]
The issue has not been fixed, the agent list is still not being rendered.

**Comment by dougfabris** [comment]
> The issue has not been fixed, the agent list is still not being rendered.

This new one #20833 will fix it

**Pull request merged**